pkg/kn/config/config_test.go

```diff
@@ -77,7 +77,8 @@ func TestBootstrapConfigWithoutConfigFile(t *testing.T) {
 	assert.Equal(t, GlobalConfig.ConfigFile(), bootstrapDefaults.configFile)
 	assert.Equal(t, GlobalConfig.PluginsDir(), bootstrapDefaults.pluginsDir)
 	assert.Equal(t, GlobalConfig.LookupPluginsInPath(), bootstrapDefaults.lookupPluginsInPath)
-	assert.Equal(t, len(GlobalConfig.SinkMappings()), 0)
+	assert.DeepEqual(t, GlobalConfig.SinkMappings(), []SinkMapping(nil))
+	assert.DeepEqual(t, GlobalConfig.ChannelTypeMappings(), []ChannelTypeMapping(nil))
```

Comment:
This change can be also reverted wrt the above comment.

Reply:
DONE